### DISCUSSION TITLE
Using 3.7.12, the dart version is reported as newer than it should be

### DISCUSSION THREAD

**Discussion by johnwiese-ms** [pingbird/puro#74] [Q&A]
I'm using Flutter 3.7.12 right now and about to being migration to Null Safety but following the docs I am stuck. docs: https://dart.dev/null-safety/migration-guide

Puro is reporting that I am in fact on 3.7.12:
<img width="394" alt="Screenshot 2024-03-25 at 8 09 54 PM" src="https://github.com/pingbird/puro/assets/54288891/6c6e6d13-60df-45fb-8226-82799c38f756">

But when I check the dart version it tells me this:

johnwiese@Johns-MacBook-Pro volleyball_parent % dart --version
Dart SDK version: 3.3.2 (stable) (Tue Mar 19 20:44:48 2024 +0000) on "macos_arm64"

but it should be telling me 2.19.6 unless I'm missing something. How do I revert the dart version to use the migration tools??

**Comment by pingbird** [comment]
the `dart` command by itself will point to the global default, in this case stable

either run `puro dart --version` inside of your project or set it as the global default with `puro use -g old`

**Reply by johnwiese-ms** [reply]
ah, ok, I assume I'll need to use the global  option so that it will allow me to use the migration tools.  Thanks!
